Makefile.common

```diff
@@ -97,4 +97,4 @@ $(FIRST_GOPATH)/bin/staticcheck:
 $(FIRST_GOPATH)/bin/govendor:
 	GOOS= GOARCH= $(GO) get -u github.com/kardianos/govendor
 
-.PHONY: all style check_license format build test vet assets tarball docker promu staticcheck $(FIRST_GOPATH)/bin/staticcheck govendor $(FIRST_GOPATH)/bin/govendor
\ No newline at end of file
+.PHONY: all style check_license format build test-short test vet tarball docker promu unused staticcheck $(FIRST_GOPATH)/bin/staticcheck $(FIRST_GOPATH)/bin/govendor
```

Comment:
why do we need both? `test-short test`

Reply:
because both targets exist

https://github.com/prometheus/prometheus/blob/0e2aa35771b23c9d65681f3b19b5a0421a206ec5/Makefile.common#L56-L63

Reply:
ooh yes , sorry I had a different understanding of what `.PHONY` targets are.